install.sh

```diff
@@ -218,12 +218,20 @@ function extract_install () {
     #extract and install
     echo_intra "Extracting ${1} ..."
     if [[ "$2" == *".zst" ]];then
-      LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Izstd -xpf ../"${2}"
+      if [[ -e /usr/bin/zstd ]]; then
+        tar -I /usr/bin/zstd -xpf ../"${2}"
+      elif ("${CREW_PREFIX}"/bin/zstd --version &> /dev/null); then
+        tar -I "${CREW_PREFIX}"/bin/zstd -xpf ../"${2}"
+      else
+        echo "Zstd is broken or nonfunctional, and some packages can not be extracted properly."
+        exit 1
+      fi
     elif [[ "$2" == *".tpxz" ]];then
-      if ! LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} pixz -h &> /dev/null; then
-        tar xpf ../"${2}"
+      if "${CREW_PREFIX}"/bin/pixz -h &> /dev/null; then
+        tar -I "${CREW_PREFIX}"/bin/pixz -xpf ../"${2}"
       else
-        LD_LIBRARY_PATH=${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX} tar -Ipixz -xpf ../"${2}"
+        echo "Pixz is broken or nonfunctional, and some packages can not be extracted properly."
```

Comment:
Why are we using pixz over system now?

Reply:
We have packages with no updates that we haven't had to rebuild.

Reply:
What was wrong with the previous tar -xvf method?